features/gba-dev/gezagsverhouding-gba.feature

```diff
@@ -0,0 +1,142 @@
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                      | waarde |
+    | indicatieCurateleRegister | true   |
+
+  Abstract Scenario: 'gezagsverhouding' veld(en) van persoon is/zijn in onderzoek
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'gezagsverhouding' gegevens
+    | naam                            | waarde                    |
+    | aanduiding in onderzoek (83.10) | <gba in onderzoek waarde> |
+    | datum ingang onderzoek (83.20)  | 20020701                  |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | inOnderzoek                     |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                                              | waarde                            |
+    | gezagInOnderzoek.aanduidingGegevensInOnderzoek    | <gba in onderzoek waarde>         |
+    | gezagInOnderzoek.datumIngangOnderzoek             | 20020701                             |
+
+
+    Voorbeelden:
+    | gba in onderzoek waarde | gezag minderjarige in onderzoek | curateleregister in onderzoek |
+    | 110000                  | true                            | true                          |
+    | 113200                  | true                            |                               |
+    | 113210                  | true                            |                               |
+    | 113300                  |                                 | true                          |
+    | 113310                  |                                 | true                          |
+
+  Abstract Scenario: 'indicatie gezag minderjarige (32.10)' is in onderzoek
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'gezagsverhouding' gegevens
+    | naam                                 | waarde                    |
+    | aanduiding in onderzoek (83.10)      | <gba in onderzoek waarde> |
+    | datum ingang onderzoek (83.20)       | 20020701                  |
+    | indicatie gezag minderjarige (32.10) | 1                         |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | indicatieGezagMinderjarige      |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                                              | waarde                    |
+    | indicatieGezagMinderjarige.code                   | 1                         |
+    | indicatieGezagMinderjarige.omschrijving           | ouder1                    |
```

Comment:
omschrijving moet 'Ouder1 heeft het gezag' zijn

Reply:
Aangepast. Ook de "sql-statements" aangepast met updates. Waardes in database komen niet overeen met publicatie.